blocks/social-share/social-share.js

```diff
@@ -1,4 +1,7 @@
-import { loadScript } from '../../scripts/aem.js';
+import { loadScript, getMetadata } from '../../scripts/aem.js';
+
+// FIXME: update date before launch
+const esdCutOff = new Date('2023-06-30');
```

Comment:
Let's have this date in the future, please, like 8 Jan 2024, as a start. having it in the past means that during QA people might report differences in counter.

Reply:
I used a date in between so we can live test. I'll change the date, no problem.